pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -3609,6 +3609,81 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			expectMatchingPodCreation(vmi)
 		})
 	})
+
+	Context("Event handling", func() {
+		var (
+			vmi *virtv1.VirtualMachineInstance
+		)
+
+		indexVirtualMachine := func(vmi *virtv1.VirtualMachineInstance) {
+			Expect(controller.vmiIndexer.Add(vmi)).To(Succeed())
+			_, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Create(context.Background(), vmi, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		pvcWith := func(requestSize, capacity, rv string) *k8sv1.PersistentVolumeClaim {
+			parsedReq, _ := resource.ParseQuantity(requestSize)
+			parsedCap, _ := resource.ParseQuantity(capacity)
```

Comment:
Use `resource.MustParse` instead, ignoring errors is not nice

Reply:
ack